app/Http/Controllers/Three/SignupsController.php

```diff
@@ -20,7 +21,7 @@ class SignupsController extends ApiController
     /**
      * Create a controller instance.
      *
-     * @param  PostContract  $posts
+     * @param PostContract  $posts
```

Comment:
Oh can you kill this `@param` here since we're not actually injecting it?

Reply:
oh yes - oops!